### PR TITLE
Game observer

### PR DESCRIPTION
**Pull request by mdesson**

### CODE REVIEW

File: src/GameObservers/GameObservers.cpp [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        row += " ";
+    }
+
+    // Add a symbol to display the end of the row
+    row += "|";
+
+    // Build the bar in the bar graph
+    for (int i = 0; i < value; i++)
+    {
+        row += "0";
```
Comment (cookfisher) [inline src/GameObservers/GameObservers.cpp]: `row += "\u25A1";  // for white box
 row += "\u25A0";  // for black box`

I find this unicode also works, if you'd like to use

**Review by cookfisher — APPROVED** [review]

File: src/GameObservers/GameObservers.cpp (hunk 1 above)
Comment (seanheinrichs) [reply src/GameObservers/GameObservers.cpp]: I think these could be cooler than the zero! I'm on board.

**Review by seanheinrichs — APPROVED** [review]
Looks good! I would be on board for changing the graph symbol but definitely good enough as is.

File: src/GameObservers/GameObservers.cpp (hunk 1 above)
Comment (mdesson) [reply src/GameObservers/GameObservers.cpp]: This is a great idea! Updated.

**Pull request merged**